pyproject.toml

```diff
@@ -11,7 +11,7 @@ python = "^3.7"
 cookiecutter = ">=1.7"
 gitpython = "^3.0"
 toml = { version = "^0.10", optional = true }
-typer = "^0.4.0"
```

Comment:
Should we change it to `<0.7` ?

Reply:
I'd call that a "Doh!" moment.  Yes, we should.  New update coming.